scripts/openstack_control_install.sh

```diff
@@ -35,3 +35,10 @@ salt -C 'I@keystone:server' cmd.run ". /root/keystonerc; heat resource-type-list
 # Install horizon dashboard
 salt -C 'I@horizon:server' state.sls horizon
 salt -C 'I@nginx:server' state.sls nginx
+
+# Install ceilometer services
+salt -C 'I@ceilometer:server' state.sls ceilometer -b 1
```

Comment:
Agreed. My comment was more to make it consistent with the rest of the deployment that deploys the backends first and then the clients. But it can tackled later.